chia/server/ws_connection.py

```diff
@@ -694,8 +695,8 @@ async def _read_one_message(self) -> Optional[Message]:
         return None
 
     # Used by the Chia Seeder.
-    def get_version(self) -> str:
-        return self.version
+    def get_chia_version_str(self) -> str:
+        return chia_version_str_from_str(self.version) or self.version
```

Comment:
I suggest keeping the existing get_version call as is -  and not adding in the `get_chia_version_str` call
This can be adjusted on the crawler side (regardless the crawler needs to be updated anyway).